docs/apis/management/user_management.yaml

```diff
@@ -116,8 +116,118 @@ paths:
                   message:
                     type: string
                     description: Error message for an unprocessable entity
-
-components: 
+    get:
+      summary: Get users
+      description: Retrieve users data for the specified account.
+      security:
+        - bearerAuth: []
+      tags:
+        - Users
+      responses:
+        "200":
+          description: Successful response containing the list of users.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  users:
```

Comment:
This looks different than the data where you create the user.
Shouldn't it be the same?

Reply:
no. it's fine